### PR TITLE
docs(tasklist): make adding procInstId to task comment configurable

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-13609

### CODE REVIEW

**Comment by tasso94** [comment]
⚠️ Rebase after #989 ist merged!

File: content/webapps/tasklist/configuration.md [hunk 1]
```diff
@@ -165,6 +165,21 @@ export default {
 
 **Note:** This does only affect the Tasklist login page. For other webapps, you need to adjust the corresponding config file as well.
 
+# Assign Process Instance Id to Task Comments
+
+By default, only the id of a task is assigned when creating a Task Comment with Tasklist. When you 
+want to query the created Task Comments later on by process instance id, this is impossible. When
```
Comment (marstamm) [inline content/webapps/tasklist/configuration.md:171]: I think the first sentence does not convey the Problem clearly. I would reword it so that it's clearer that the process instance ID is missing.

```
When creating a Task Comment, the process instance ID is not assigned by default. 
Queries for comments by process instance ID will not include those comments.
```

Comment (tasso94) [reply content/webapps/tasklist/configuration.md:171]: 👍

**Review by marstamm — APPROVED** [review]
